generators/server/templates/src/main/java/package/security/_PersistentTokenRememberMeServices.java

```diff
@@ -92,11 +92,11 @@
 
     private static final int TOKEN_VALIDITY_SECONDS = 60 * 60 * 24 * TOKEN_VALIDITY_DAYS;
 
-    private static final int UPGRADED_TOKEN_VALIDITY_SECONDS = 5;
+    private static final long UPGRADED_TOKEN_VALIDITY_MILLIS = 5_000l;
```

Comment:
This number isn't big enough to add the underscore, it will just confuse novice users. I would rather just use `5000L`

Reply:
OK, sure, that makes sense.